test_runner/regress/test_combocid.py

```diff
@@ -5,12 +5,7 @@
 
 def do_combocid_op(neon_env_builder: NeonEnvBuilder, op):
     env = neon_env_builder.init_start()
-    endpoint = env.endpoints.create_start(
-        "main",
-        config_lines=[
-            "shared_buffers='1MB'",
```

Comment:
I see a bunch of `"shared_buffers='1MB'"` removals, what's the default  `shared_buffers` if LFC is not enabled?

Reply:
1MB is the default value if LFC is not enabled